client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -0,0 +1,206 @@
+import FormTextarea from 'components/forms/form-textarea';
+import HeaderCake from 'components/header-cake';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import wrapSettingsForm from 'my-sites/site-settings/wrap-settings-form';
+import categories from './categories';
+
+class PodcastDetails extends Component {
+	renderExplicitContent() {
+		const { fields, handleSelect, isRequestingSettings, translate } = this.props;
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="podcasting_explicit">{ translate( 'Explicit Content' ) }</FormLabel>
+				<FormSelect
+					id="podcasting_explicit"
+					name="podcasting_explicit"
+					onChange={ handleSelect }
+					value={ fields.podcasting_explicit || 'no' }
+					disabled={ isRequestingSettings }
+				>
+					<option value="no">{ translate( 'No' ) }</option>
+					<option value="yes">{ translate( 'Yes' ) }</option>
+					<option value="clean">{ translate( 'Clean' ) }</option>
+				</FormSelect>
+			</FormFieldset>
+		);
+	}
+
+	renderSaveButton() {
+		const { handleSubmitForm, isRequestingSettings, isSavingSettings, translate } = this.props;
+		return (
+			<Button
+				compact={ true }
+				onClick={ handleSubmitForm }
+				primary={ true }
+				type="submit"
+				disabled={ isRequestingSettings || isSavingSettings }
+			>
+				{ isSavingSettings ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+			</Button>
+		);
+	}
+
+	renderTextField( { FormComponent = FormInput, key, label } ) {
+		const { fields, isRequestingSettings, onChangeField } = this.props;
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor={ key }>{ label }</FormLabel>
+				<FormComponent
+					id={ key }
+					name={ key }
+					type="text"
+					value={ fields[ key ] || '' }
+					onChange={ onChangeField( key ) }
+					disabled={ isRequestingSettings }
+				/>
+			</FormFieldset>
+		);
+	}
+
+	renderTopicSelector( key ) {
+		const { fields, handleSelect, isRequestingSettings } = this.props;
+		return (
+			<FormSelect
+				id={ key }
+				name={ key }
+				onChange={ handleSelect }
+				value={ fields[ key ] || 0 }
+				disabled={ isRequestingSettings }
+			>
+				<option value="0">None</option>
+				{ map( toPairs( categories ), ( [ category, subcategories ] ) => {
+					// The keys for podcasting in iTunes use &amp;
+					const catKey = category.replace( '&', '&amp;' );
+					return [
+						<option key={ catKey } value={ catKey }>
+							{ category }
+						</option>,
+						...map( subcategories, subcategory => {
+							const subcatKey = catKey + ',' + subcategory.replace( '&', '&amp;' );
+							return (
+								<option key={ subcatKey } value={ subcatKey }>
+									{ category } » { subcategory }
+								</option>
+							);
+						} ),
+					];
+				} ) }
+			</FormSelect>
+		);
+	}
+
+	renderTopics() {
+		const { translate } = this.props;
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="podcasting_category_1">{ translate( 'Podcast Topics' ) }</FormLabel>
```

Comment:
I intentionally did not rename this setting to `Podcasting Topics`.  We are configuring the _podcasting settings_ on this site, and one of those settings is the topic of _this site's podcast_.  This makes sense to me, but we can definitely change it later if someone else has a better idea.